assignment1.txt

```diff
@@ -0,0 +1,7 @@
+Jared Wasserman
+
+1) A compiler turns the code written in C (or other high level languages) into Assembly code (which is then translated by Assembler into machine code) so it can be executed.
```

Comment:
right, more specifically, a compiler takes your code written in C, removes the white space, places this into a parse tree, optimizes it, and then takes that parse tree and translates it into assembly code.